### DISCUSSION TITLE
make it easier to increment measurable checkmark widget

### DISCUSSION THREAD

**Discussion by JackElsey** [iSoron/uhabits#1469] [Feature Requests]
It would be nice if it were easier to increase measurable goals via the checkmark widget. When I tap the checkmark widget for one of my measurable goals this is the interface I get. Almost 99% of the time I just want to increase the total by one, but to do so I need to squint at the current value (rendered in a small font and hard to read in sunlight), then carefully tap out the next higher integer with the numeric keyboard. 

![image](https://user-images.githubusercontent.com/19174587/185810525-e3db2725-c99b-4530-b1f4-afcabae5daf8.png)

If I touch the "1" in the white window the keyboard disappears, causing the white window to shift down and the selected value to jump forward to "2".

![image](https://user-images.githubusercontent.com/19174587/185810910-31b37c8b-85a7-4187-8b90-6e9f9e3b8881.png)

The alternative input method I'd suggest are "+" and "-" buttons that increase or decrease the value by a certain user-specified increment.

**Comment by rajat2012** [comment]
I also agree with him...
It will be great if we can add increment value by pressing "+" and incase we want to decrease value we can use "-" button. It will be convenient for those who have multiple checkmark for one habit.

If possible please implement this feature.
Thanks in advanced...

**Comment by YalandHong** [comment]
This seems to be a duplicate of #951 and #994
